File: Array/SortColors.py
def sort_colors(arr):
	colors = [0, 0, 0]
	for num in arr:
		colors[num] += 1

	j = 0
	for i in range(len(colors)):
		while colors[i] > 0:
			arr[j] = i
			j += 1
			colors[i] -= 1
	return arr

File: Array/SortColors.js
function colorSort(arr){
	var colors = {0:0, 1:0, 2:0};
	var order = [];
	for(i=0; i<arr.length; i++){
		colors[arr[i]] += 1;
	}

	for(i=0; i<3; i++){
		while(colors[i] > 0){
			order.push(i);
			colors[i]--;
		}
	}
	console.log(order);
}

colorSort([0, 2, 0, 1, 2, 1]);